test/helpers/cilium.go

```diff
@@ -328,7 +328,13 @@ func (s *SSHMeta) PolicyDelAll() *CmdRes {
 
 // PolicyDel deletes the policy with the given ID from Cilium.
 func (s *SSHMeta) PolicyDel(id string) *CmdRes {
-	return s.ExecCilium(fmt.Sprintf("policy delete %s", id))
+	res := s.ExecCilium(fmt.Sprintf(
+		"policy delete %s | grep Revision: | awk '{print $2}'", id))
```

Comment:
Does this command have an option for JSON output? I want to move away from using `grep` and `awk` as it makes the output we parse dependent upon ordering as opposed to a key-value mapping which JSON provides.